tools/log-collector/helper.go

```diff
@@ -271,8 +266,8 @@ func checkLabelExist(givenLabel, toCheckInLabel map[string]string) (exist bool)
 	return exist
 }
 
-// filterObjectsOnLabel filter objects on the basis of Labels
-func filterObjectsOnLabel(allObjects unstructured.UnstructuredList) (objects unstructured.UnstructuredList) {
+// filterTvkResourcesByLabel filter objects on the basis of Labels
+func filterTvkResourcesByLabel(allObjects unstructured.UnstructuredList) (objects unstructured.UnstructuredList) {
```

Comment:
pass arg `allObjects` by pointer... no need of return arg

Reply:
resolved